migrations/sqls/20190726234757-programs-up.sql

```diff
@@ -0,0 +1,155 @@
+/* Replace with your SQL commands */
+CREATE TABLE programs (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  programName varchar(255),
+  programDescription varchar(1028)
+);
+
+INSERT INTO programs (programName, programDescription) VALUES
+  ("Preventive Health", "Preventive Health"),
+  ("Nutrition / CRYS / SNAP", "Nutrition / CRYS / SNAP"),
+  ("Immigration", "Immigration"),
+  ("Financial Education / Coaching", "Financial Advising / Coaching"),
+  ("Workers' Rights", "Workers' Rights"),
+  ("Youth Groups", "Youth Groups"),
+  ("Family Support", "Family Support Services"),
+  ("Community Engagement and Organizing", "Community Engagement and Organizing")
+;
+
+ALTER TABLE services
+  ADD COLUMN programId INT,
+  ADD FOREIGN KEY programIdFk(programId) REFERENCES programs(id);
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Immigration"
+) WHERE serviceName = "Citizenship";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Immigration"
+) WHERE serviceName = "Family Petition";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Workers' Rights"
+) WHERE serviceName = "Workers' Rights and Safety";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Immigration"
+) WHERE serviceName = "DACA";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Youth Groups"
+) WHERE serviceName = "Youth Groups";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Nutrition / CRYS / SNAP"
+) WHERE serviceName = "SNAP";
+
+UPDATE services SET programId = (
+  SELECT id from programs WHERE programName = "Preventive Health"
+) WHERE serviceName = "Chronic Disease Testing";
```

Comment:
"Chronic Diseases Testing" doesn't appear to be on the intake form anymore. At least not the June 2019 copy.

Reply:
See https://docs.google.com/document/d/1fF-8KjJG16Yk7I_qG2X4FpydZRgFtM2yiyHmiLNjSjI/edit which is the most recent form that Vicky just sent me. The name has now changed to Chronic Diseases Screening, but it still exists.